sdl_android/src/main/java/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -2265,19 +2265,20 @@ public void handleMessage(int flags, byte[] packet){
 			}
 			if(buffer == null){
 				Log.e(TAG, "Unable to assemble message as buffer was null/not started");
```

Comment:
I don't think we really need this log message. For the sake of clean code it should be removed and the this made into a single if statement, or null check if with a return